### DISCUSSION TITLE
Issue when I installed offline moose in HPC

### DISCUSSION THREAD

**Discussion by lvkas521424** [idaholab/moose#27447] [Q&A General] [answered]
When I make -j 8 in  ~/moose/test/ , it's a fatal error that gcc7.5.0 can't recognize the  include filesystem

@GiudGiud and @milljm I'm sorry to mention you again. Firstly, I'm so sorry that my improper operation has had a negative impact on your and the users' work. After reinstalling gcc10.4.0 、mpich4.2.0、PETSc、libmesh, I have solved the problem I encountered earlier. Therefore, I think maybe it is necessary for you to inform users of the requirements for GCC in different versions. For example, when I was using the latest version, if you want to make in /test/ file, it required a language requirement of c++17, which forced me to reinstall GCC versions higher than 8.0.

Jiahuilv

**Comment by loganharbour** [comment, marked as answer]
Sorry for the troubles @lvkas521424. I've opened the issue https://github.com/idaholab/moose/issues/27449 to get this resolved.

I dug around quite a bit and do see that we reference GCC 7.5 as a valid compiler in many cases. Thanks for reaching out to us and making this request, it's definitely important and hopefully we can avoid these frustrations for other users.
